sdk/cosmos/azure-cosmos/azure/cosmos/_base.py

```diff
@@ -673,3 +676,16 @@ def validate_cache_staleness_value(max_integrated_cache_staleness):
     int(max_integrated_cache_staleness)  # Will throw error if data type cant be converted to int
     if max_integrated_cache_staleness <= 0:
         raise ValueError("Parameter 'max_integrated_cache_staleness_in_ms' can only be a positive integer.")
+
+
+def _stringify_auto_scale(offer):
+    auto_scale_params = None
+    max_throughput = offer.auto_scale_max_throughput
+    increment_percent = offer.auto_scale_increment_percent
+    if max_throughput is not None:
+        auto_scale_params = {"maxThroughput": max_throughput}
+        if increment_percent is not None:
+            auto_scale_params["autoUpgradePolicy"] = {"throughputPolicy": {"incrementPercent": increment_percent}}
```

Comment:
This is still silently ignoring `increment_percent` if `max_throughput` is not supplied. If the server response to this situation is inadequate, then this should raise a local error.

Reply:
We raise an error for if `increment_percent` is supplied without `max_throughput` so this is fine and we would want increment percent to not exist if we are not supplying the max throughput.

Reply:
This comment is still unaddressed.

Reply:
@tjprescott I'm unsure what you mean by this or what's missing here - if the user only supplied increment_percent and not max_throughput, then the client returns an error to the user (line 709 in this same file) which is the behavior you described to begin with and the behavior we are aiming for. Could you expand on what's missing, or what type of flaw you are talking about?

Reply:
@simorenoh @JericHunter all of that validation is provided in the setter and the SDK will emit a client error if it is violated. Thus, there is no reason to duplicate validation here. It should simply plumb everything through.